packages/openssl.rb

```diff
@@ -1,13 +1,29 @@
 require 'package'
 
 class Openssl < Package
-  version '1.0.1e'
-  binary_url ({
-    i686: 'https://dl.dropboxusercontent.com/s/w6y84tusor5xz5f/openssl-1.0.1e-chromeos-i686.tar.gz?token_hash=AAGQ2xjngbnzme2CKee7Mz5WvkylBtFy1rwUzWDVNuOQ_Q&dl=1',
-    x86_64: 'https://dl.dropboxusercontent.com/s/384awniosicvm12/openssl-1.0.1e-chromeos-x86_64.tar.gz?token_hash=AAH4sdqkNnhIFU-uPdrpqddsi8UU0vWe_gwkplUBM_40MQ&dl=1'
-  })
-  binary_sha1 ({
-    i686: 'cadea32ec770c4b44d565b7e5fdf96a469a05757',
-    x86_64: '3cf4defb11fc2fccce77736d0f4559e56d9d7e05'
-  })
+  version '1.0.2j'
+
+#  chromeos wget can't do proper ssl negotiation with this server
+#  source_url 'https://www.openssl.org/source/openssl-1.0.2j.tar.gz'
+#  so use their ftp server.  
+  source_url 'ftp://openssl.org/source/openssl-1.0.2j.tar.gz'
+  source_sha1 'bdfbdb416942f666865fa48fe13c2d0e588df54f'
+
+  depends_on 'perl'
+
+  def self.build
+	system "./config","--prefix=/usr/local","shared","zlib-dynamic"
+	system "make" #depend
+  end
+
+  def self.install
+    system "make"
+    system "sudo make DESTDIR=/usr/local/ssl install"
+    system 'cd /usr/local/ssl && \                                # make sure cert.pem exists
+    	    sudo wget http://curl.haxx.se/ca/cacert.pem && \
+    	    sudo mv cacert.pem cert.pem'                                # && \
+    	    #ln -s /usr/local/ssl/bin/openssl /usr/local/bin/openssl'    # override the system version (in case of a version mismatch)
+                                                                  	# the caveat here is /usr/local/bin should come before /usr/bin in $PATH (the default)
```

Comment:
I'm not sure why we need cert.pem here.  At least, isn't it possible to make these lines to echo commands?